lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -157,6 +166,25 @@ class ReportUIFeatures {
     this._copyAttempt = false;
   }
 
+  _setupMetricDescriptionToggleElements() {
+    this.metricDescriptionToggleEl =
+      /** @type {HTMLInputElement} */ (this._dom.find('.lh-metrics-toggle input', this._document));
+    this.metricAuditGroup = this._dom.find('.lh-audit-group--metrics', this._document);
+  }
+
+  _setupMetricDescriptionToggleListeners() {
```

Comment:
i definitely didnt expect this..

click on the newly expanded descriptions to make them go away?

this UX doesn't make sense to me

Reply:
IMO this would only make sense if the toggle is scoped to just the metric title.

Reply:
![image](https://user-images.githubusercontent.com/4071474/56931914-97724d80-6a96-11e9-91d9-bea41d4ad059.png)
https://docs.google.com/document/d/15L5iaWo4ieouegpSlaqGGWpA13JnTv73xnpxuVhzkBg/edit?ts=5cc77eb3

Reply:
silly that u cant link to doc comments

Reply:
aiight. i'm not feeling this UX at all, so I raised a concern there: https://docs.google.com/a/google.com/document/d/15L5iaWo4ieouegpSlaqGGWpA13JnTv73xnpxuVhzkBg/edit?disco=AAAAC6njMnI


> silly that u cant link to doc comments

![image](https://user-images.githubusercontent.com/39191/56932799-fd140900-6a99-11e9-80e5-c5ad83b0097b.png)